setup-knative/action.yaml

```diff
@@ -75,6 +75,10 @@ runs:
           # between versions, so have to fetch for each component.
           if [ ${{ inputs.version }} == "latest" ]; then
             REAL_KNATIVE_VERSION=$(curl -L -s https://api.github.com/repos/knative/${REPO}/releases/latest | jq -r '.tag_name')
+          elif [[ ${{ inputs.version }} == *".x"* ]]; then
```

Comment:
nit: can you add a comment to say this matches trailing `.x`

should it be matching anything after the `.x`? should we drop the trailing `*`?
otherwise, 1.7.xa would match, but you only drop the `.x` for the INPUT.

Reply:
if there is anything after the x like `v1.2.xsdfds` that will not match and most likely the action will fail

Reply:
thanks for the review, please let me know if the comment are ok/good @k4leung4